signac/contrib/project.py

```diff
@@ -1811,7 +1834,7 @@ def repair(self, fn_statepoints=None, index=None, job_ids=None):
                 raise
         if index is not None:
             for doc in index:
-                self._sp_cache[doc["signac_id"]] = doc["statepoint"]
+                self._sp_cache[doc["signac_id"]] = doc["sp"]
```

Comment:
Is it worth adding a test for this? Not sure how often `repair` is really used in the wild, but it would be nice if it was tested.

Reply:
I think our plan for 2.0 is to remove most index-related logic, so I don't think we need to test this behavior.